packages/material-ui/src/FormControl/useFormControl.d.ts

```diff
@@ -4,7 +4,14 @@ import { FormControlProps } from './FormControl';
 // shut off automatic exporting
 export {};
 
-type ContextFromPropsKey = 'disabled' | 'error' | 'hiddenLabel' | 'margin' | 'required' | 'variant';
+type ContextFromPropsKey =
+  | 'disabled'
+  | 'error'
+  | 'fullWidth'
```

Comment:
I'm not sure this is correct or even necessary, but it seems like a type update would be good to do for anything that returns the updated context object.